mlx/traceability_exception.py

```diff
@@ -32,6 +32,10 @@ def __init__(self, message, docname=''):
         '''
         super(TraceabilityException, self).__init__(message)
         self.docname = docname
+        self.message = message
+
+    def __str__(self):
+        return str(self.message)
```

Comment:
i would expect this method is derived from Exception or some other higher super class?

Reply:
Yes, I would as well, but issue in #85 made me think this is the error (which was confirmed it is not). It will require a bit more complex debugging